ArduPlane/AP_Arming.cpp

```diff
@@ -4,20 +4,6 @@
 #include "AP_Arming.h"
 #include "Plane.h"
 
-const AP_Param::GroupInfo AP_Arming_Plane::var_info[] = {
```

Comment:
I haven't tested, but I wouldn't be surprised if this broke all of the plane ARMING parameters that people already had, since this significantly alters the table structure. The minimum change is to retain the table and only remove the rudder sub entry (and leave a comment indicating the entry can't be used).